src/MySqlConnector/Logging/Log.cs

```diff
@@ -402,26 +402,26 @@ internal static partial class Log
 	[LoggerMessage(EventIds.FoundSessionToCleanUp, LogLevel.Debug, "Pool {PoolId} found session {SessionId} to clean up")]
 	public static partial void FoundSessionToCleanUp(ILogger logger, int poolId, string sessionId);
 
-	[LoggerMessage(EventIds.HasServerRedirectionHeader, LogLevel.Trace, "Session {SessionId} has server redirection header {Header}")]
-	public static partial void HasServerRedirectionHeader(ILogger logger, string sessionId, string header);
+	[LoggerMessage(EventIds.HasServerRedirectionHeader, LogLevel.Trace, "{poolPrefix}Session {SessionId} has server redirection header {Header}")]
+	public static partial void HasServerRedirectionHeader(ILogger logger, string poolPrefix, string sessionId, string header);
```

Comment:
The log field names are significant, and are used for "structured logging" within Microsoft.Extensions.Logging; see https://learn.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-8.0#log-message-template.

We can't just change them for better code reuse (thinking of them like string concatenation).